src/app/legacy/containers/BulletedListItem/index.jsx

```diff
@@ -10,7 +10,14 @@ const withClickHandler = (Component, clickHandler) => props =>
   <Component {...props} onClick={clickHandler} />;
 
 const BulletedListItemContainer = ({ blocks, onClick }) => {
-  const contentBlocks = blocks.map(block => block.model.blocks).flat();
+  const contentBlocks = blocks
+    .map(block => {
+      if (block.model.blocks !== undefined) {
+        return block.model.blocks;
+      }
+      return block;
+    })
```

Comment:
This could be simplified a little bit.

Worth noting that you don't need to include `!== undefined` in the if statement - JS will return true if `block.model.blocks` is truthy - not null or undefined or 0

You could also collapse this into a simple OR statement (but please test this out just incase it doesn't work!

```suggestion
    .map(block =>  block.model.blocks || blocks)
```

Reply:
Ah thanks for this! Yes I remember thinking that .map function looked a bit clunky. It looks much cleaner now.